include/ignition/physics/ConstructEmpty.hh

```diff
@@ -73,6 +73,30 @@ class ConstructEmptyModelFeature : public virtual Feature
   };
 };
 
+/////////////////////////////////////////////////
+/// \brief This feature constructs an empty nested model and return its pointer
+/// from the given world.
+class ConstructEmptyNestedModelFeature : public virtual Feature
+{
+  public: template <typename PolicyT, typename FeaturesT>
+  class Model : public virtual Feature::Model<PolicyT, FeaturesT>
+  {
+    public: using ModelPtrType = ModelPtr<PolicyT, FeaturesT>;
+
+    /// \brief Construct an empty model and attach a given name to it.
+    /// \return
+    ///   The ModelPtrType of the constructed model.
+    public: ModelPtrType ConstructEmptyModel(const std::string &_name);
```

Comment:
Similar to my comment [here](https://github.com/ignitionrobotics/ign-physics/pull/228/files#r599303356) I would recommend naming this function `ConstructEmptyNestedModel` to make the nested model's relationship to the parent model more explicit.

Reply:
Changed in https://github.com/ignitionrobotics/ign-physics/pull/229/commits/f0a849285efb0a501e89030cd318eb2ce7b7b175